arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/copy/ExtCodeCopySection.java

```diff
@@ -98,6 +100,7 @@ public ExtCodeCopySection(Hub hub) {
               .makeWithTrm(accountBefore, accountBefore, rawAddress, doingDomSubStamps);
 
       this.addFragment(accountReadingFragment);
+      commonValues.setTracedException(TracedException.OUT_OF_GAS_EXCEPTION);
```

Comment:
The setting would disappear if we do include the `COPY` family in the high-level processing in `CommonFragmentValues`